### PR TITLE
Small language edits Patch 1

### PR DESCRIPTION
**Pull request by rhondae**
----
:books: Documentation preview :books:: https://repository-service-tuf--543.org.readthedocs.build/en/543/

### CODE REVIEW

**Comment by rhondae** [comment]
Small language changes and the addition of one video link.

File: docs/source/index.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ Talks
 =====
 
 * `PackagingCon 2023 <https://cfp.packaging-con.org/2023/talk/997FQK/>`_:
-  “Our stuff” - how to protect users from package compromise with RSTUF (October 2023, comming soon).
+  “Our stuff” - how to protect users from package compromise with RSTUF (October 2023, coming soon).
```
Comment (kairoaraujo) [inline docs/source/index.rst:80]: Adding the recording that is already available :) 

```suggestion
  `“Our stuff” - how to protect users from package compromise with RSTUF <https://www.youtube.com/live/mYzqeS2d96A?si=cqQUH5J5wDPrOQFb&t=3801>`_.
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -94,8 +94,8 @@ Posts
 Mentions
 ========
 
-* Open Source Summit EU 2023: at OpenSSF day Omkhar Arasaratnam, OpenSSF General Manager, mentions
-  Repository Service for TUF.
+* `"Open Source Summit EU 2023: at OpenSSF day Omkhar Arasaratnam, OpenSSF General Manager, mentions
+  Repository Service for TUF" <https://www.youtube.com/watch?v=WWY6SWaYhxI>`
 
   .. image:: ./_static/images/rstuf_openssf_day_openssf_eu_2023.jpg
     :width: 200
```
Comment (kairoaraujo) [inline docs/source/index.rst:101]: I think we can remove this image as we have the link to the video.
```suggestion
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -94,8 +94,8 @@ Posts
 Mentions
 ========
 
-* Open Source Summit EU 2023: at OpenSSF day Omkhar Arasaratnam, OpenSSF General Manager, mentions
-  Repository Service for TUF.
+* `"Open Source Summit EU 2023: at OpenSSF day Omkhar Arasaratnam, OpenSSF General Manager, mentions
+  Repository Service for TUF" <https://www.youtube.com/watch?v=WWY6SWaYhxI>`
```
Comment (kairoaraujo) [inline docs/source/index.rst:98]: Nit: link direct to the RSTUF mention.

```suggestion
* `"Open Source Summit EU 2023: at OpenSSF day Omkhar Arasaratnam, OpenSSF General Manager, mentions
  Repository Service for TUF" <https://youtu.be/WWY6SWaYhxI?si=bT34fgeQquWWh8ga>`_
```

**Review by kairoaraujo — COMMENTED** [review]
Hi @rhondae
Thank you for your contribution, it is amazing! 🎉 

I made some comments, let me know if you need more details.

**Review by kairoaraujo — APPROVED** [review]

**Pull request merged**